hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/ScheduledTaskStatisticsImpl.java

```diff
@@ -23,20 +23,24 @@
 import java.io.IOException;
 import java.util.concurrent.TimeUnit;
 
+import static com.hazelcast.internal.cluster.Versions.V3_10;
+
 public class ScheduledTaskStatisticsImpl
         implements ScheduledTaskStatistics, TaskLifecycleListener {
```

Comment:
Needs to implement `Versioned`

Reply:
I always forget this one, thanks @Donnerbart